packages/plugin-core/src/commands/ShowPreview.ts

```diff
@@ -94,31 +102,138 @@ export const extractNoteIdFromHref = (data: {
   return out;
 };
 
-enum LinkType {
+export enum LinkType {
   WIKI = "WIKI",
+  ASSET = "ASSET",
   WEBSITE = "WEBSITE",
   MARKDOWN = "MARKDOWN",
   UNKNOWN = "UNKNOWN",
 }
 
 const classifyLink = ({ href }: NoteViewMessage["data"]): LinkType => {
-  if (href && href.startsWith("vscode-webview")) {
+  if (
+    href &&
+    (href.startsWith("dendron://") ||
```

Comment:
we only support this format for cross vault links to dendron notes, not assets. 
its interesting to think how we would link to assets in other vaults but we'd want to think about how we want to support that. 

we're actually moving away from regular markdown links alltogether and handling assets according to this instead -> [[Note References over Note Types|dendron://private/task.markdown.2021.11.18.note-references-over-note-types]]

Reply:
Rolled back the change of 'dendron://'